test/CMakeLists.txt

```diff
@@ -78,9 +78,15 @@ file(GLOB test_srcs
   test_*.cu
 )
 
+file(GLOB catch2_test_srcs
+  RELATIVE ${CUB_SOURCE_DIR}/test
+  CONFIGURE_DEPENDS
+  catch2_test_*.cu
```

Comment:
Do we really need the `catch2_` prefix?

Reply:
After everything is settled - I don't think we need one. Right now we have `test_` prefix for existing infrastructure, which is going to coexist for a while.

Reply:
Yeah sorry for the noise, I should have checked that we have more than 3 test files ;)